test/test_circular_buffer.jl

```diff
@@ -1,6 +1,10 @@
 using DataStructures
-using Base.Test
-
+if VERSION >= v"0.5.0-dev+7720"
```

Comment:
It seems to have been set up so that you can also run at least some of the test files independently. For example, if I'm hacking on one particular structure, it's nice to be able to quickly run the tests for just that structure.